Replicator/tests/ReplicatorAPITest.cc

```diff
@@ -889,8 +889,8 @@ TEST_CASE_METHOD(ReplicatorAPITest, "Calling c4socket_ method after STOP", "[C][
 
     waitForStatus(kC4Offline);
 
-    _numCallbacksWithLevel[kC4Connecting] = 0;
-    waitForStatus(kC4Connecting);
+    _numCallbacksWithLevel[kC4Offline] = 0;
+    waitForStatus(kC4Offline);
```

Comment:
Same as above, why there are two wait for offline? Please add a comment there if it's intended.

Reply:
Ah, you could be right, something I missed. I will investigate and update the test where necessary.